packages/turf-boolean-within/index.ts

```diff
@@ -138,20 +138,18 @@ function isMultiPointOnLine(multiPoint: MultiPoint, lineString: LineString) {
 }
 
 function isMultiPointInPoly(multiPoint: MultiPoint, polygon: Polygon) {
-  var output = true;
-  var oneInside = false;
-  var isInside = false;
-  for (var i = 0; i < multiPoint.coordinates.length; i++) {
+  let output = true;
+  let isInside = false;
+  for (let i = 0; i < multiPoint.coordinates.length; i++) {
     isInside = booleanPointInPolygon(multiPoint.coordinates[i], polygon);
     if (!isInside) {
       output = false;
       break;
     }
-    if (!oneInside) {
-      isInside = booleanPointInPolygon(multiPoint.coordinates[i], polygon, {
-        ignoreBoundary: true,
-      });
-    }
+
+    isInside = booleanPointInPolygon(multiPoint.coordinates[i], polygon, {
+      ignoreBoundary: true,
+    });
```

Comment:
This change is a no-op (I guess?) but its confusing as part of the yarn upgrade. It also might be a step away from a better version where we can skip the second booleanPointInPolygon? Not really sure what the original intent was of the oneInside boolean.

Reply:
Me either. Yeah, it basically evaluated to ```if (true) ...```

Was an issue that didn't seem to justify its own PR, though happy to separate it out.